view/frontend/templates/js/replacejs.phtml

```diff
@@ -879,9 +932,7 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
                     $.get(settings.create_order_url, params)
                         .done(function (data) {
 
-                            var boltRestricted = !!data.restrict;
-
-                            toggleCheckoutIfNeeded(boltRestricted);
+                            BoltState.cartRestricted = !!data.restrict;
 
                             // Stop if Bolt checkout is restricted
                             if (boltRestricted) {
```

Comment:
is `boltRestricted` defined for line 938 (887 in original)?

Reply:
Further down there's a similar sequence where `boltRestricted` becomes `BoltState.cartRestricted`, I'm guessing similar should happen here.

Reply:
Thank you Nathaniel, good catch. This code is about payment only and it doesn't work now, but we should be consistent. Fixed.